docs/standard/whats-new/whats-new-in-dotnet-standard.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "What's new in the .NET Standard"
+ms.custom: "updateeachrelease"
+ms.date: "10/31/2017"
+ms.prod: ".net"
+ms.topic: "article"
+ms.technology: dotnet-standard
+ms.##devlang: dotnet
+author: "rpetrusha"
+ms.author: "ronpet"
+manager: "wpickett"
+---
+
+# What's new in the .NET Standard
+
+The .NET Standard is a formal specification that defines a versioned set of APIs which must be available on .NET implementations that comply with that version of the standard. The .NET Standard is targeted at library developers. It aims to guarantee the functionality that is available to a library that targets a version of the .NET Standard on each .NET implementation. In other words, a library that targets a version of the .NET Standard can be used on any .NET Framework, .NET Core, or Xamarin implementation that supports that version of the standard.
+
+The most recent version of the .NET Standard is 2.0. It is included with the .NET Core 2.0 SDK, as well as with Visual Studio 2017 Version 15.3 with the .NET Core workload installed.
+
+## Supported .NET implementations
+
+The .NET Standard 2.0 is supported by the following .NET implementations:
+
+- .NET Core 2.0
+- .NET Framework 4.6.1
+- Mono 5.4
+- Xamarin.iOS 10.14
+- Xamarin.Mac 3.8
+- Xamarin.Android 8.0
+- Universal Windows Platform 10.0.16299
+
+## What's new in the .NET Standard 2.0
+ 
+The .NET Standard 2.0 includes the following new features:
+
+**A vastly expanded set of APIs**
+
+Through version 1.6, the .NET Standard included a comparatively small subset of APIs. Among those excluded were many APIs that were commonly used in the .NET Framework or Xamarin. This complicates development, since it requires that developers find suitable replacements for familiar APIs when they develop applications and libraries that target multiple .NET implementations. The .NET Standard 2.0 addresses this limitation by adding over 20,000 more APIs than were available in .NET Standard 1.6, the previous version of the standard. For a list of the APIs that have been added to the .NET Standard 2.0, see [.NET Standard 2.0 vs 1.6](https://raw.githubusercontent.com/dotnet/standard/master/docs/versions/netstandard2.0_diff.md). 
+
+Some of the more noteworthy additions to the <xref:System> namespace alone in .NET Standard 2.0 include the following:
+
```

Comment:
I would replace with something like "<xref:System> namespace additions in .NET Standard 2.0 include:".